modules/openapi-generator/src/main/resources/typescript-axios/modelObjectEnum.mustache

```diff
@@ -1,14 +1,14 @@
-export const {{classname}}{{enumName}} = {
```

Comment:
does this work when there are multiple enums defined in a model?

Reply:
Yes. 
I generated codes using the schema below, and correct codes are generated.

### openapi schema
```yaml
openapi: 3.0.3
info:
  title: Enum
  description: Enum
  version: 1.0.0
paths:
  /items:
    get:
      responses:
        200:
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Response'
  /generic:
    get:
      responses:
        200:
          description: OK
          content:
            application/json:
              schema:
                type: object
                properties:
                  m1:
                    type: string
                    enum: [ a, b, c ]
                  m2:
                    type: number
                    enum: [ 1, 2, 3 ]
components:
  schemas:
    Response:
      type: object
      properties:
        enum1:
          type: string
          enum: [ a, b, c ]
        enum2:
          type: string
          enum: [ e, d, f ]
```

### `stringEnums = false`
```typescript
export interface InlineResponse200 {
    'm1'?: InlineResponse200M1Enum;
    'm2'?: InlineResponse200M2Enum;
}

export const InlineResponse200M1Enum = {
    A: 'a',
    B: 'b',
    C: 'c'
} as const;

export type InlineResponse200M1Enum = typeof InlineResponse200M1Enum[keyof typeof InlineResponse200M1Enum];
export const InlineResponse200M2Enum = {
    NUMBER_1: 1,
    NUMBER_2: 2,
    NUMBER_3: 3
} as const;

export type InlineResponse200M2Enum = typeof InlineResponse200M2Enum[keyof typeof InlineResponse200M2Enum];

export interface Response {
    'enum1'?: ResponseEnum1Enum;
    'enum2'?: ResponseEnum2Enum;
}

export const ResponseEnum1Enum = {
    A: 'a',
    B: 'b',
    C: 'c'
} as const;

export type ResponseEnum1Enum = typeof ResponseEnum1Enum[keyof typeof ResponseEnum1Enum];
export const ResponseEnum2Enum = {
    E: 'e',
    D: 'd',
    F: 'f'
} as const;

export type ResponseEnum2Enum = typeof ResponseEnum2Enum[keyof typeof ResponseEnum2Enum];
```


### `stringEnums = true`

```typescript
export interface InlineResponse200 {
    'm1'?: InlineResponse200M1Enum;
    'm2'?: InlineResponse200M2Enum;
}

export enum InlineResponse200M1Enum {
    A = 'a',
    B = 'b',
    C = 'c'
}
export enum InlineResponse200M2Enum {
    NUMBER_1 = 1,
    NUMBER_2 = 2,
    NUMBER_3 = 3
}

export interface Response {
    'enum1'?: ResponseEnum1Enum;
    'enum2'?: ResponseEnum2Enum;
}

export enum ResponseEnum1Enum {
    A = 'a',
    B = 'b',
    C = 'c'
}
export enum ResponseEnum2Enum {
    E = 'e',
    D = 'd',
    F = 'f'
}
```